ws-nextjs-app/cypress.config.ts

```diff
@@ -12,19 +22,75 @@ export default defineConfig({
       const appEnv = config.env.APP_ENV;
       const env = config.env[appEnv];
 
+      const { parsed } = dotenv.config({
+        path: `./envConfig/${config.env.APP_ENV}.env`,
+      });
+
+      const appConfig = parsed as Record<string, string>
+      const envVars = Object.keys(appConfig).reduce((vars, key) => {
+        vars[key] = JSON.stringify(appConfig[key]);
+        return vars;
+      }, {});
+
       config.baseUrl = env.baseUrl;
 
       // Debugging console logs to see running config
       /* eslint-disable no-console */
       console.log('\n\n\n\n\n');
       console.log('Cypress running config:');
-      // console.log('SMOKE:', config.env.SMOKE);
+      console.log('SMOKE:', config.env.SMOKE);
       console.log('APP_ENV:', config.env.APP_ENV);
       console.log('Base URL:', config.baseUrl);
       // console.log('UK:', config.env.UK);
       console.log('\n\n\n\n\n');
       /* eslint-enable no-console */
 
+      const options = {
+        webpackOptions: {
+          resolve: {
+            extensions: ['.ts', '.tsx', '.js', '.jsx'],
+            alias: {
+              '#src': resolvePath('../src'),
+              '#app': resolvePath('../src/app'),
+              '#psammead': resolvePath('../src/app/legacy/psammead'),
+              '#lib': resolvePath('../src/app/lib/'),
+            },
+          },
+          module: {
+            rules: [
+              {
+                test: /\.(ts|tsx|js|jsx)$/,
+                exclude: /node_modules/,
+                use: {
+                  loader: 'babel-loader',
+                  options: {
+                    presets: [
+                      '@babel/preset-env',
+                      '@babel/preset-react',
+                      '@babel/preset-typescript',
+                    ],
+                  },
+                },
+              },
+            ],
+          },
+          plugins: [
+            // @ts-expect-error - TODO: fix types
+            new MomentTimezoneInclude({ startYear: 2010, endYear: 2025 }),
```

Comment:
Is the typescript TODO going to be addressed in this PR? Also even though it's still only September, could/should the end year be updated to at least 2026?